tests/std/tests/P0067R5_charconv/test.cpp

```diff
@@ -460,7 +464,17 @@ constexpr errc inv_arg = errc::invalid_argument;
 constexpr errc out_ran = errc::result_out_of_range;
 
 template <typename T>
-void test_integer_from_chars() {
+_CONSTEXPR23 bool test_integer_from_chars() {
+    string hundred_zeroes              = string(100, '0');
+    string hundred_zeroes_and_11       = hundred_zeroes + "11"s;
+    string minus_hundred_zeroes        = "-"s + hundred_zeroes;
+    string minus_hundred_zeroes_and_11 = "-"s + hundred_zeroes_and_11;
+
+    string hundred_ones                  = string(100, '1');
+    string hundred_ones_and_atatat       = hundred_ones + "@@@"s;
+    string minus_hundred_ones            = "-"s + hundred_ones;
+    string minus_hundred_ones_and_atatat = "-"s + hundred_ones_and_atatat;

```

Comment:
Suggestion: These can be `const`, and the `hundred_zeroes` and `hundred_ones` can be direct-initialized.